src/_main_/utils/pagination/__init__.py

```diff
@@ -0,0 +1,26 @@
+from django.core.paginator import Paginator, EmptyPage
+from _main_.utils.common import serialize_all
+
+DEFAULT_LIMIT = 50
+def paginate(queryset, page, limit):
+    try:
+        if not limit:
+            limit = DEFAULT_LIMIT
+        p = Paginator(queryset, limit)
```

Comment:
```suggestion
        paginator = Paginator(queryset, limit or DEFAULT_LIMIT)
```

Reply:
Name your variables well to avoid confusion between page and paginator.  When someone sees `p` they could guess either.  Make it less ambiguous